### PR TITLE
Add rpi-otp-private-key clarification

### PR DESCRIPTION
**Pull request by ysi-camerona**
See https://github.com/raspberrypi/rpi-eeprom/issues/607. Although the device-specific private key OTP area exists on all Raspberry Pi boards, the `rpi-otp-private-key` script only supports BCM2711 and BCM2712-based boards.

### CODE REVIEW

**Review by timg236 — APPROVED** [review]

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (lurch) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:175]: You've missed the 'M' out of BCM2711 and BCM2712 :wink: 
https://www.raspberrypi.com/documentation/computers/processors.html

Comment (ysi-camerona) [reply documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc]: Good catch, thank you! I've force-pushed an amended commit rather than adding a typo fix commit, hope that is okay. 🙂

**Comment by ysi-camerona** [comment]
This PR requires maintainer approval before it can be merged. @timg236 or @lurch: do you know who can approve and merge?

**Comment by nathan-contino** [comment]
Looking at this now.

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:175]:
```suggestion
==== Program a key into OTP with `rpi-otp-private-key`

NOTE: The `rpi-otp-private-key` script only works on devices that use the Broadcom xref:processors.adoc#bcm2711[BCM2711] or xref:processors.adoc#bcm2712[BCM2712]) processors.
```

**Review by nathan-contino — APPROVED** [review]
LGTM after a minor fix.

File: documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/raspberry-pi-industrial.adoc:176]:
```suggestion
NOTE: The `rpi-otp-private-key` script only works on devices that use the Broadcom xref:processors.adoc#bcm2711[BCM2711] or xref:processors.adoc#bcm2712[BCM2712] processors.
```

**Pull request merged**
